templates/ansible/facts.erb

```diff
@@ -103,16 +118,21 @@ def main():
     module.exit_json(**return_value)
 
 
+<% if object.collection -%>
+<%= lines(emit_link('collection', build_url(object.collection), object)) -%>
```

Comment:
Is there a way that this could have been some sort of override instead of a separate field?

Reply:
I definitely think it should be! Currently, Puppet, Chef, Ansible all handle collection URLs this way. I'd definitely like to make that change, but maybe wait for a different PR?

Reply:
Filing an issue for this: https://github.com/GoogleCloudPlatform/magic-modules/issues/453